scripts/exgfs_wave_post_pnt.sh

```diff
@@ -566,9 +554,9 @@ source "${USHgfs}/preamble.sh"
   if [ "$wavenproc" -gt '1' ]
   then
     if [ ${CFP_MP:-"NO"} = "YES" ]; then
-      ${wavempexec} -n ${wavenproc} ${wave_mpmd} cmdmprogbouy
+      ${wavempexec} -n "${wavenproc}" "${wave_mpmd}" cmdmprogbuoy
     else
-      ${wavempexec} ${wavenproc} ${wave_mpmd} cmdfile.bouy
+      ${wavempexec} "${wavenproc}" "${wave_mpmd}" cmdfile.buoy
     fi
     exit=$?
   else
```

Comment:
Can be in a later PR, but this should be updated to use `ush/run_mpmd.sh`

Reply:
I think I will pass on this since the job will soon be MPI'd.